src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -462,14 +462,13 @@ public void parse_repoAliases_sameResult() throws ParseException, HelpScreenExce
         Assert.assertEquals(repoAliasCliArguments, reposAliasCliArguments);
     }
 
-    @Test
-    public void parse_invalidRepoLocation_emptyRepoConfigurationList()
+    @Test (expected = InvalidLocationException.class)
+    public void parse_invalidRepoLocation_throwsInvalidLocationException()
```

Comment:
`RepoSense.getRepoConfigurations` throws  invalidLocException only when there are no valid repo locations provided right (regardless of the number of "RepoLocations" being thrown in)?

 If so, perhaps the test name can be changed to something like `parse_noValidRepoLocation_throws...()`.

Reply:
sure

Reply:
Yes `RepoSense.getRepoConfigurations` throws invalidLocationException only when no valid locations provided AND the number of repolocations in argument >= 1. If no repo location supplied at all, it will be handled by ArgsParser at an earlier stage.